build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/NiaFlavor.kt

```diff
@@ -21,13 +21,16 @@ enum class NiaFlavor(val dimension: FlavorDimension, val applicationIdSuffix: St
 
 fun configureFlavors(
     commonExtension: CommonExtension<*, *, *, *, *, *>,
-    flavorConfigurationBlock: ProductFlavor.(flavor: NiaFlavor) -> Unit = {}
+    flavorConfigurationBlock: ProductFlavor.(flavor: NiaFlavor) -> Unit = {},
 ) {
     commonExtension.apply {
-        flavorDimensions += FlavorDimension.contentType.name
+        FlavorDimension.values().forEach {
```

Comment:
Better to have specific variable name instead of it:     

        FlavorDimension.values().forEach { dimension ->
            flavorDimensions += dimension.name
        }

Reply:
Thank you, I updated lambda variable name based on its type name.